src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -1602,6 +1602,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
             Return trivia.IsElastic()
         End Function
 
+        Public Function IsPragmaDirective(trivia As SyntaxTrivia, ByRef isDisable As Boolean, ByRef isActive As Boolean, ByRef errorCodes As SeparatedSyntaxList(Of SyntaxNode)) As Boolean Implements ISyntaxFacts.IsPragmaDirective
+            Return trivia.IsPragmaDirective(isDisable, isActive, errorCodes)
```

Comment:
❔ Is this a recursive call to the same method?

Reply:
Nope, calling an extension method. I followed the same pattern used in all implementations in this file.